src/setup/setup_framework.py

```diff
@@ -8,21 +9,81 @@ def get_options() -> dict:
     return setup_non_interative(args)
 
 def setup_interative() -> dict:
-    return interactor.terminal_input()
+    options = {}
+    # Populates the dictionary with the necessary options
+    options['maker'] = interactor.maker_type_input()
+    filepath, label_name = interactor.based_mode_input()  
+    options['filepath'] = filepath
+    
+    # Check if is in normal mode or based-on mode
+    # Refers to the normal mode
+    if filepath == "":
+        options['samples'] = interactor.samples_input()
+        options['attributes'] = interactor.attributes_input()
+        options['classes'] = interactor.classes_input()
+        measures = interactor.measures_input()
+
+    # Refers to the based-on mode
+    if filepath != "":
+        df, label = read_based_on_dataset(filepath, label_name)
+        options['samples', 'attributes', 'classes'] = extract_properties(df, 
+                                                                      label)
+        # TODO: Extract Measures from the real dataset
```

Comment:
In order to extract these measures it is necessary a robust complexity extraction. The module is highly dependent of `ECoL` package, but that's not the ideal, due to the fact that the framework should be ready to insert measures other than Complexity Measures. This can be adressed in [#20](https://github.com/SteffanoP/cbdgen-framework/issues/20)

Reply:
Addressed in #20 objectives